src/coreclr/gc/gc.cpp

```diff
@@ -20433,12 +20461,9 @@ void gc_heap::update_collection_counts ()
     }
 }
 
+#ifndef USE_REGIONS
```

Comment:
nit - you could put `extend_soh_for_no_gc` next to this method since they are for the same purpose